genologics/entities.py

```diff
@@ -373,14 +369,13 @@ class BooleanDescriptor(StringDescriptor):
     """An instance attribute containing a boolean value
     represented by an XMl element.
     """
-
     def __get__(self, instance, cls):
-        instance.get()
-        node = self.get_node(instance)
-        if node is None:
-            return None
-        else:
-            return node.text.lower() == 'true'
+        text = super(BooleanDescriptor, self).__get__(instance, cls)
+        if text is not None:
+            return text.lower() == 'true'
+
+    def __set__(self, instance, value):
+        super(BooleanDescriptor, self).__set__(instance, str(value).lower())
```

Comment:
I'm not a fan of allowing any non-boolean value to be set as a BooleanDescriptor. 
However, you could argue that Clarity will simply reject the update. Should we also handle it here ?